docs/modules/plugins/pages/plugin-rest-api.adoc

```diff
@@ -276,6 +276,30 @@ Then server `$hostname` supports secure protocols that $rule `$protocols`
 Then server `vividus-test-site.herokuapp.com` supports secure protocols that contain `TLSv1.2,TLSv1.3`
 ----
 
+=== Wait for expected HTTP status code in response
+
+Waits for a specified amount of times until HTTP response code is not equal to what is expected. In case if after of all retries the expected code is not returned exception will be thrown.
```

Comment:
```suggestion
Waits for the specified number of times until HTTP response code is equal to the expected one. In case if the expected code is not returned after all retries, the assertion error will be recorded.
```